spring-integration-gemfire/src/main/java/org/springframework/integration/gemfire/store/GemfireMessageStore.java

```diff
@@ -29,9 +32,6 @@
 import org.springframework.util.Assert;
 import org.springframework.util.PatternMatchUtils;
 
-import com.gemstone.gemfire.cache.Cache;
-import com.gemstone.gemfire.cache.Region;
-
```

Comment:
Yep! Just has fixed import settings on Windows. Made them the same like on Mac:
![imports](https://cloud.githubusercontent.com/assets/1071590/11669202/77911200-9dc9-11e5-8a32-254a122320bf.png)